Resources/Prototypes/Entities/Clothing/Shoes/specific.yml

```diff
@@ -119,6 +119,9 @@
   - type: SpeedModifierContactCapClothing
     maxContactSprintSlowdown: 0.7
     maxContactWalkSlowdown: 0.7
+  - type: Tag
+    tags:
+    - ClothingShoesGaloshes
```

Comment:
This change should be properly commented.

Something like `# Harmony change. Tag component is needed for ducky galoshes` on the `type` line.